Developer_readme.md

```diff
@@ -0,0 +1,237 @@
+     |  \
+     |   |prompt_stats (stores new prompt stats)
+     |   |sk.acrobatics_pronecheckf (toggled)
+     |   |valid.check_life
+     |   |sk.onprompt_beforelifevision_do (-> svo.aiprompt())
+     |
+     |send_in_the_gnomes
+     |  \
+     |   |lifevision.validate
+     |   |signals.after_lifevision_processing
+     |      \
+     |       |sk.onprompt_beforeaction_do / prompttrigger
+     |       |cnrl.checkwarning
+     |       |sp_checksp
+     |   |make_gnomes_work (curing commands)
+     |
+     |signals.after_prompt_processing
+     |  \
+     |   |custom prompt
+     |   |cnrl.dolockwarning
+
+
+# How to's
+
+## How to add a new affliction
+1. add it in **svo (actions dictionary)** in the dict table, with the appropriate functions and curing logic.
+1. add it in empty cures on **svo (setup, misc, empty, funnies, dor) > Empty cure handling** script.
+3. add new diagnose trigger for it on **svo (aliases, triggers)**.
+5. add to herb cures/restoration/smoke/focus/humour/etc tables (whatever that is applicable) on **svo (trigger functions) > Main trigger functions** (the comment lines `-- normal herbs`, `-- normal smokes`. `-- focus`, etc can help you find your way).
+7. add to generic cures (passive cures or cures that happen in blackout) (generic_cures_data in **Main trigger functions** on **svo (trigger functions)**)
+8. add its definition on svof-serverside integration (sstosvoa on dict).
+9. (Optional) add triggers receiving and losing affliction. Note: This should only be necessary for afflictions that are not shown on GMCP for some reason (best example I can think of is Pariah's latency), the system already handles gaining/removing aff through GMCP so no need to add triggers for that on normal circumstances.
+10. (Optional) check failure conditions and add them (in case it is an affliction that is not trackeable on gmcp). You can add the logic on the dict entry or on triggers, wherever applicable.
+<details>
+    <summary>This is an example of how an affliction entry is structured: </summary>
+    
+```lua
+  paralysis = {
+    herb = {
+      isadvisable = function() -- add all the conditions necessary for the system to consider that you have been afflicted by this here
+        return (affs.paralysis) or false -- means if you are paralysed already, it won't double paralysis
+      end,
+
+      oncompleted = function() -- affliction got cured, signal svo to remove it from the affs list and that herb balance has been lost since you just ate the herb to cure this, add other logic if necessary
+        svo.rmaff('paralysis')
+        svo.lostbal_herb()
+        svo.killaction(svo.dict.checkparalysis.misc)
+      end,
+
+      eatcure = {'bloodroot', 'magnesium'}, -- tells svo what kind of cure to make this go away
+      onstart = function() svo.eat(svo.dict.paralysis.herb) end, -- svo started the curing procedure, tell it what it should do (in this case, it will eat the specified cure above)
```

Comment:
```suggestion
      onstart = function() svo.eat(svo.dict.paralysis.herb) end, -- svo started the curing procedure because `isadvisable()` above said it should, tell it what it should do (in this case, it will eat the specified `eatcure`)
```